src/Symfony/Component/Messenger/Transport/AmqpExt/Connection.php

```diff
@@ -206,6 +206,9 @@ public function channel(): \AMQPChannel
             }
 
             $this->amqpChannel = $this->amqpFactory->createChannel($connection);
+            if (isset($this->connectionCredentials['prefetch_count'])) {
+                $this->amqpChannel->setPrefetchCount($this->connectionCredentials['prefetch_count']);
+            }
```

Comment:
This is an extra feature unrelated to the main purpose of this PR, right?

Reply:
It's related to AMQP rejection process. By default, RabbitMQ uses a prefetch of 3. If you requeue a message, it comes behind the prefetched messages but not in the queue itself. If consumption speed is not important, you can choose a prefetch of 1, then the messages go back to the queue.